oat-sa/extension-tao-testqti-previewer
# Feat/AUT-741/inject rich passage styles in itemData

Related to: https://oat-sa.atlassian.net/browse/AUT-741

Inject rich passage stylesheets as item stylesheets in ItemData with absolute URL

need branches:
https://github.com/oat-sa/extension-tao-itemqti/tree/feature/CON-57%2Frich-passage-authoring
https://github.com/oat-sa/extension-tao-mediamanager/tree/feature/CON-57%2Frich-passage-authoring

How to test:
1. create Rich passages with own styles (use Style editor)
2.  add these Rich passages in item
3. save item
4. refresh page
5. preview item, Rich passage styles should be applied
